packages/safe-tools-client/app/templates/schedule.hbs

```diff
@@ -35,6 +35,20 @@
         >
           <div class={{cn itemCssClass safe-tools__current-network-option=(eq this.network.symbol network.symbol)}} data-test-selected-network>{{network.name}}</div>
         </Boxel::Select>
+      {{else if this.wallet.isConnected}}
+        <div class="safe-tools__current-network">
+          <span class="safe-tools__current-network-text">{{this.network.networkInfo.name}}</span>
+          <Boxel::IconButton
+            @icon="info"
+            aria-label="Current Network"
+            style={{css-var
+              boxel-icon-button-width=0
+              boxel-icon-button-height=0
+              icon-color='var(--boxel-light)'
+            }}
+            data-hover="You can change the network from your wallet. Supported networks: {{(array-join this.network.supportedNetworksName ',')}}"
```

Comment:
Unfortunately this may not be true for wallet connect 2.0, I don't think there's an action to take now, but just a heads up to keep this on our radar

Reply:
We can remove it once wallet connect 2.0 is implemented.